test/lib/server-common.ts

```diff
@@ -41,10 +41,10 @@ export interface ServerOptions
 export abstract class Server
 {
 	public port: number | null = null;
+	public receivedCookies: Array<string> = new Array<string>();
```

Comment:
[minor]: It is not a good idea to use constructors as types. 
- Some constructors, such as Object, Function, or Array, are very broad and do not provide much type safety or information. Using them as types can make the code less clear and more prone to bugs.
- TypeScript has built-in types for primitive values, such as number, string, boolean, etc. Using constructors like Number, String, or Boolean as types can lead to unexpected behavior and type errors.

```suggestion
	public receivedCookies: string[] = [];
```